source/agora/consensus/data/Transaction.d

```diff
@@ -183,6 +236,14 @@ public struct Output
         PublicKey key = PublicKey(point[]);
         return key;
     }
+
+    /// Support for sorting
+    public int opCmp ( const typeof(this) rhs ) const nothrow @safe @nogc
+    {
+        if (this.lock != rhs.lock)
+            return this.lock < rhs.lock ? -1 : 1;
+        return this.value.opCmp(rhs.value);
+    }
```

Comment:
Is the rule that the first output is a refund kept?
This rule has been added in the PR below. I'd like to check, please.
https://github.com/bosagora/agora/pull/1664
When freezing 40,000 BOA from a UTXO containing 50,000 BOA, two outputs are generated. However, 40,000 BOA is not frozen, and other outputs may be frozen.

Reply:
No, the rule is now that a single output may be less than the minimum freeze amount as long as there are more than one outputs.

Reply:
Are you ok now with the change?